CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 - Deserializing SentryEvents with Decodable (#4724)
 - Remove internal unknown dict for Breadcrumbs (#4803) This potentially only impacts hybrid SDKs.
+- Add abnormal mechanism to SentrySession (#4842)
```

Comment:
If this helps with some customer-facing thing, it should be worded in a way they will understand. 
```suggestion
- Enhancements to help report ANR rates (#4842)
```

Reply:
I like your suggestion 👍  Now that I think about it again, I think it's nothing the users should care about. I removed it from the changelog.